### ISSUE TITLE
SceneView "New / GameObject" item does not respect multi-selection

### ISSUE THREAD

**Issue by deanljohnson** [AdamsLair/duality#647]
### Summary

Right clicking on the SceneView panel and selecting "New / GameObject" only creates an child object for the object clicked on when multiple objects are selected. I'd expect all objects selected to receive a new child object.

### How to reproduce [Bugs]

Select multiple objects within the SceneView panel. Then right click on one of them and select "New / GameObject". Only the object you right clicked on will receive a child.

**Comment by ilexp** [comment]
Yep, that sounds like something we should change. Would expect the same behavior, but apparently it was never implemented that way.

Is multi-selection respected when creating new components for existing objects?
